### PR TITLE
Add Bitget

### PR DESCRIPTION
**Pull request by Yang-Nankai**
Bitget is a cryptocurrency exchange platform that offers trading services for digital assets, including spot and futures trading. The platform provides users with advanced trading tools, social trading features, and a secure environment for buying, selling, and managing cryptocurrencies. Bitget offers 2FA for the account login verification process. The website is ranked 2,335 on [[Similarweb](https://pro.similarweb.com/#/digitalsuite/websiteanalysis/overview/website-performance/*/999/1m?webSource=Total&key=bitget.com)](https://pro.similarweb.com/#/digitalsuite/websiteanalysis/overview/website-performance/*/999/1m?webSource=Total&key=bitget.com). The website provides documentation on how to turn on 2FA.

### CODE REVIEW

File: entries/b/bitget.com.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "Bitget": {
+    "domain": "bitget.com",
+    "tfa": [
+      "call",
+      "sms",
+      "totp"
+    ],
+    "documentation": "https://www.bitget.com/academy/an-advanced-security-guide-on-bitget",
```
Comment (fletcherdodd) [inline entries/b/bitget.com.json]:
```suggestion
    "documentation": "https://www.bitget.com/support/articles/12560603808391",
    "recovery": "https://www.bitget.com/support/articles/12560603808511",
```

File: img/b/bitget.com.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path fill="#0088E5" d="M2.2 9.2c1.7-1 3.5-1.7 5.2-2.4 3-1.3 5.9-2.5 8.9-3.6.6-.2 1.6-.4 1.7.4 0 1.3-.3 2.5-.4 3.7l-1.4 8c-.1.8-1.2 1.3-2 .7l-5-3.4c-.6-.6 0-1.4.4-1.8 1.4-1.4 3-2.6 4.2-4 .4-.9-.7-.1-1 0-2 1.4-3.8 2.8-5.8 4-1 .5-2.2 0-3.3-.3-.9-.4-2.3-.7-1.5-1.3Z"/></svg>
```
Comment (fletcherdodd) [inline img/b/bitget.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="#00e0ff" viewBox="0 0 23.73 25.52"><path d="M10.56 9.16h6.35l6.5 6.46c.42.42.42 1.1 0 1.53l-8.34 8.38H8.53l1.98-1.92 7.26-7.22-7.17-7.22m2.57 7.19H6.82L.32 9.9c-.42-.42-.42-1.1 0-1.52L8.65 0h6.54l-1.98 1.92-7.26 7.22 7.17 7.22"/></svg>
```

File: entries/b/bitget.com.json (hunk 1 above)
Comment (Yang-Nankai) [reply entries/b/bitget.com.json]: Thanks for your correction!

File: img/b/bitget.com.svg (hunk 2 above)
Comment (Yang-Nankai) [reply img/b/bitget.com.svg]: Thank you for your modification, but may I ask what is the basis for your modification? ❤️ @fletcherdodd

**Review by fletcherdodd — APPROVED** [review]

File: img/b/bitget.com.svg (hunk 2 above)
Comment (fletcherdodd) [reply img/b/bitget.com.svg]: No worries @Yang-Nankai! As per [the guidelines, point 2](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#guidelines) talks about optimising your file as much as possible to reduce the file size. I usually run my SVG files through 3 different optimisation tools (a couple are mentioned in contributing doc) just to ensure it as small as possible. 😇

**Pull request merged**